src/initializers/diamorphosis.ts

```diff
@@ -2,6 +2,40 @@ import * as diamorphosis from 'diamorphosis';
 import { OrkaOptions } from '../typings/orka';
 import { defaultTo, isBoolean } from 'lodash';
 
+export const producerConfigOverrides = config => {
+  const {
+    KAFKA_PRODUCER_BROKERS,
+    KAFKA_PRODUCER_CERTIFICATES_KEY,
+    KAFKA_PRODUCER_CERTIFICATES_CERT,
+    KAFKA_PRODUCER_CERTIFICATES_CA,
+    KAFKA_PRODUCER_SASL_USERNAME,
+    KAFKA_PRODUCER_SASL_PASSWORD
+  } = process.env;
+
+  if (!KAFKA_PRODUCER_BROKERS) {
+    return {
+      ...config.kafka.producer,
+      brokers: config.kafka.brokers,
+      certificates: config.kafka.certificates,
+      sasl: config.kafka.sasl
+    };
+  }
+
+  return {
+    ...config.kafka.producer,
```

Comment:
🔧 What if someone has overwritten some values in his config.js?
You will overwrite them with the below code.